app/viewmodels/IndexPageHelper.scala

```diff
@@ -196,44 +195,21 @@ class IndexPageHelper @Inject()(p: views.html.components.p,
     }
   }
 
-  private def returnMessageIfAmountMoreThanZero(amount: BigDecimal, msgKeyToApply: String)(implicit messages: Messages): Option[String] = {
-    if(amount > 0) {
-      val formattedAmount = if (amount.isWhole()) amount else "%,.2f".format(amount)
-      Some(messages(msgKeyToApply, formattedAmount))
-    } else None
-  }
-
-  private def returnEstimatedMessageIfHasEstimatedCharges(amount: BigDecimal, isEstimate: Boolean, msgKeyToApply: String)(implicit messages: Messages): Option[String] = {
-    if(amount > 0) {
-      val msgKey = if(isEstimate) s"$msgKeyToApply.estimated" else msgKeyToApply
-      Some(messages(msgKey, amount))
-    } else None
-  }
-
   private def returnMessageIfOtherUnrelatedPenalties(isUnrelatedPenalties: Boolean, msgKey: String)(implicit messages: Messages): Option[String] = {
-    if(isUnrelatedPenalties) {
+    if (isUnrelatedPenalties) {
       Some(messages(msgKey))
     } else None
   }
 
-  private def returnEstimatedVATMessageIfMoreThanZero(vatInterest: BigDecimal, isEstimatedVAT: Boolean, msgKeyToApply: String)(implicit messages: Messages): Option[String]={
-   if(vatInterest > 0) {
-    (isEstimatedVAT,vatInterest.isWhole() ) match {
-      case(true,true) => Some(messages(s"$msgKeyToApply.estimated", vatInterest))
-      case(true,false) => Some(messages(s"$msgKeyToApply.estimated", "%,.2f".format(vatInterest)))
-      case(false,false) => Some(messages(s"$msgKeyToApply", "%,.2f".format(vatInterest)))
-      case(false,true) => Some(messages(s"$msgKeyToApply", vatInterest))
```

Comment:
Similar naming observations here 'vatInterest' and 'isEstimatedVAT' and the method name 'returnEstimatedMessageIfInterestMoreThanZero' - which is being used as a generic one to display values and estimated values.

Reply:
Sorted!